epoxy-composeinterop-maverickssample/src/androidTest/java/com/airbnb/epoxy/composeinterop/maverickssample/MultiKeyComposeInteropFragmentTest.kt

```diff
@@ -0,0 +1,152 @@
+package com.airbnb.epoxy.composeinterop.maverickssample
+
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.compose.ui.test.onAllNodesWithText
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performClick
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class MultiKeyComposeInteropFragmentTest {
+    @get:Rule
+    val composeTestRule = createAndroidComposeRule<MainActivity>()
+
+    @Test
+    fun testComposableInInteropUsingMutableStateUpdatesEvenWithoutKey() {
+        for (i in 0..10) {
+            val textToAssert = "withoutInteropKey mutableState: $i, recomposingCount: ${i + 1}"
+            waitUntil(textToAssert)
+            checkIfTextIsDisplayed(textToAssert)
+
+            composeTestRule.onNodeWithText(textToAssert).performClick()
+        }
+
+        waitUntil("withoutInteropKey mutableState: 11, recomposingCount: 12")
```

Comment:
synced offline about how using waitUntil isn't ideal. 

`Any condition that depends on Android's measure or draw (that is, measure or draw external to Compose) should use more general concept such as waitUntil()`